pkg/agent/config/service_config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import v1 "k8s.io/api/core/v1"
+
+const (
+	ServiceTypeExternalIP v1.ServiceType = "ExternalIP"
```

Comment:
If there is no such type in K8s, I don't think this is a valid type. How was it tested?

Reply:
ExternalIP is not a Service type in K8s, an ExternalIP could be added to any type of Service, like ClusterIP, NodePort or LoadBalancer. Due to our API of OfClient, we need another value of ServiceType to call `InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, nodeLocalExternal bool, svcType v1.ServiceType, nested bool) error`. I know that this is not a good idea to add a serviceType value like this. Do you know we could pass type `LoadBalancer` here?